.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: CI (Python, Docker Build)
```

Comment:
I don't see a `Python` thing in here but not a huge deal

Reply:
shoot thanks yeah, that's copy pasta

Reply:
fixed